lib/transform.js

```diff
@@ -0,0 +1,81 @@
+const vegadataflow = require('../vendor/vega-dataflow.js');
+
+
+var tx = vegadataflow.transforms,
+    changeset = vegadataflow.changeset;
+
+
+export function aggregate(fields, operations, data) {
```

Comment:
Same here. Think this would be equivalent for 
```
export aggregate = (fields, operations, data) => {...}
// or 
const aggregate = (fields, operations, data) => {...}
export aggregate
```